src/main/java/org/jenkinsci/tools/configcloner/handler/TransferHandler.java

```diff
@@ -132,7 +129,12 @@ private CommandResponse send(
                     destination, xmlString, this.updateCommandName(), destJob
             );
 
-            if (rsp.succeeded()) return response.returnCode(0);
+            if (rsp.succeeded()) {
+                return response.returnCode(0);
+            } else {
+                System.err.println("Updating xml failed with stderr:" + rsp.stderr());
+                System.out.println("Updating xml failed with stdout:" + rsp.stdout());
```

Comment:
As an error message, this should go to standard error (despite it was reported by underlying CLI command standard output). Also, for testability sake, do not print directly to `System.err/out`, use `response` instead.

Reply:
updated